contracts/IConversionPathFinder.sol

```diff
@@ -3,9 +3,9 @@ pragma solidity 0.6.12;
 
 import "./token/interfaces/IReserveToken.sol";
 
-/*
-    Conversion Path Finder interface
-*/
+/**
+ * @dev Conversion Path Finder interface
```

Comment:
the reason we didn't add @dev before was that it wasn't supported by gitbooks - if I'm not mistaken.
so we just need to make sure that one works.

Reply:
Ah, super weird. It's pretty common. In any case, it'd be easy to revert later on